### PR TITLE
Implement new design for Manage Music Assets

### PR DESCRIPTION
**Pull request by jvmsangkal**
### a. DB Changes:
- N/A

### b. Package Changes:
- none

### c. Implementation Changes:
- Showing number of checked assets
- Changed position of autorelease buttons

### a. Ticket link/Request Feature link
- [Tunes-121](https://freedom.myjetbrains.com/youtrack/issue/Tunes-121)

### b. Tests Done
- [Functional Test in Asana](https://app.asana.com/0/116336656053273/116880250851982)

### CODE REVIEW

File: app/scripts/components/music_assets/index.js [hunk 1]
```diff
@@ -234,23 +234,39 @@ var React = require('react'),
...
                             </div>
                         </div>
                         <div className="row">
                             <div className="col s12">
                                 <div className="card">
                                     <div className="card-content">
+                                        <div id="auto-release-row" className="row" ref="auto_release">
+                                            <div className="col s12 m6 l6">
+                                                <h6>
+                                                    {this.state.checked.length} items selected
```
Comment (sibayanjasper) [inline app/scripts/components/music_assets/index.js]: Include in translation

File: app/scripts/components/claims/index.js [hunk 2]
```diff
@@ -31,9 +31,9 @@ var React = require('react'),
             var results =  ClaimsStore.getClaims();
 
             _(results).forEach(function (e) {
-                e.release_date = moment(new Date(e.release_date)).format('MMM DD YYYY');
-                e.insert_date = moment(new Date(e.insert_date)).format('MMM DD YYYY');
-                e.update_date = moment(new Date(e.update_date)).format('MMM DD YYYY');
+                e.release_date = e.release_date ? moment(new Date(e.release_date)).format('MMM DD YYYY') : '';
+                e.insert_date = e.insert_date ? moment(new Date(e.insert_date)).format('MMM DD YYYY') : '';
+                e.update_date = new Date(e.update_date)? moment(new Date(e.update_date)).format('MMM DD YYYY') : '';
```
Comment (sibayanjasper) [inline app/scripts/components/claims/index.js]: Why do this have `new Date` while the other two don't?

File: app/scripts/components/music_assets/check_asset.js [hunk 3]
```diff
@@ -10,8 +10,12 @@ CheckAsset = React.createClass({
 
         document.getElementById('check_all').checked = MusicAssetStore.isAllChecked(e.target.checked, e.target.id);
 
-        document.getElementById('auto-release').disabled = !e.target.checked && !(MusicAssetStore.getChecked().length + (e.target.checked ? 1 : -1));
-        document.getElementById('not-auto-release').disabled = !e.target.checked && !(MusicAssetStore.getChecked().length + (e.target.checked ? 1 : -1));
+        if (!e.target.checked && !(MusicAssetStore.getChecked().length + (e.target.checked ? 1 : -1))) {
```
Comment (sibayanjasper) [inline app/scripts/components/music_assets/check_asset.js]: Could you explain this?
I guess `e.target` is the current row and `MusicAssetStore.getChecked()` returns all selected rows. Is that right?

Comment (jvmsangkal) [inline app/scripts/components/music_assets/check_asset.js]: added the explanation as comment in the code

File: app/scripts/components/music_assets/index.js [hunk 4]
```diff
@@ -234,23 +235,39 @@ var React = require('react'),
...
+                                                            <i className="material-icons">close</i> No
+                                                        </a>
+                                                    </h6>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div id="total-assets-row" className="row">
+                                            <div className="col s12 right-align">
+                                                <h5>
+                                                    Total Assets: {this.state.total}
```
Comment (sibayanjasper) [inline app/scripts/components/music_assets/index.js]: Include in translation (also include the Yes and No above).
Also, try using `this.props.intl.formatMessage({id: <string id>})`.

**Pull request merged**